src/main/java/io/reactivesw/category/infrastructure/validator/CategoryNameValidator.java

```diff
@@ -38,15 +42,24 @@ public static void validateNull(CategoryDraft draft) {
     if (draft.getName() == null
         || draft.getName().getLocalized() == null
         || draft.getName().getLocalized().isEmpty()) {
-      LOG.debug("CategoryView name can not be null");
-      throw new ParametersException("CategoryView name can not be null");
+      LOG.debug("CategoryView name can not be null.");
+      throw new ParametersException("CategoryView name can not be null.");
     }
+    Predicate<Map.Entry<String, String>>
+        entryPredicate =
+        (entry) -> StringUtils.isEmpty(entry.getValue()) || StringUtils.isEmpty(entry.getKey());
```

Comment:
I did not say it is wrong, is not need. 
And the indentation of IDEA is not good enough.